service/account/test/src/test/java/org/eclipse/kapua/service/account/test/AccountLocatorConfiguration.java

```diff
@@ -72,10 +78,15 @@ protected void configure() {
                 bind(AccountEntityManagerFactory.class).toInstance(AccountEntityManagerFactory.getInstance());
                 bind(AccountService.class).toInstance(new AccountServiceImpl());
                 bind(AccountFactory.class).toInstance(new AccountFactoryImpl());
+
+                //Initialize JAXB context
+                Multibinder.newSetBinder(binder(), ClassProvider.class).addBinding().to(TestJAXBClassProvider.class);
+                bind(JAXBContextProvider.class).to(JAXBContextProviderImpl.class);
             }
         };
 
         Injector injector = Guice.createInjector(module);
+        XmlUtil.setContextProvider(injector.getInstance(JAXBContextProvider.class));
         mockedLocator.setInjector(injector);
```

Comment:
JAXB is now initialized within the context of the DI context initialization. As per production code, XmlUtil is temporarily statically initialized here - proper DI to be introduced in the future